examples/cmd/testapp/cmd/log.go

```diff
@@ -34,6 +34,17 @@ var LogCmd = &cobra.Command{
 	},
 }
 
+var TerminalLogCmd = &cobra.Command{
+	Use:     "terminal-log",
+	Short:   "Command to log some information",
```

Comment:
I'm not sure if we actually use / care about these command instantiation params - if we do, it could be good to add some more description of what the terminal log command is. I needed the PR description to understand what this change is doing, I could see it being confusing without that context.

Reply:
Even this being just a sample application, yeah you're right, I will put a better description here.